packages/flutter/lib/src/widgets/table.dart

```diff
@@ -348,39 +348,44 @@ class _TableElement extends RenderObjectElement {
   @override
   RenderTable get renderObject => super.renderObject as RenderTable;
 
-  // This class ignores the child's slot entirely.
-  // Instead of doing incremental updates to the child list, it replaces the entire list each frame.
-
```

Comment:
@Hixie 
It’s weird to deal with it this way. We should not recreate(drop and new create) the cell elements because their keys and types have not changed, right?
For example, We should not rebuild element and RO in the following situation，which you concerning
https://github.com/flutter/flutter/pull/69670/files/60b124b9a86717f4e0aeaf203e23c8d0c1e3944e#r565894797
before rebuild:  `[Text('A'), Text('B'), Text('C'), Text('D'), Text('E'), ]` with (column:3, row:2) 
after update: `[Text('a'), Text('b'), Text('c'), Text('d'), Text('e'), ]` with (column:2, row:3)

Reply:
Ignore the child slot is the root cause of the related issue.

Reply:
i agree that it would be better to not replace the elements in that case